bots/openprescribing/openprescribing.py

```diff
@@ -81,6 +82,13 @@ def deploy_live_now(message):
     reset_or_deploy_timer(0, message)
 
 
+@respond_to('op staging deploy (.*)', re.IGNORECASE)
+def deploy_branch_to_staging(message, branch):
+    safe_execute(
+        deploy, hosts=HOSTS, environment='staging', branch=branch)
```

Comment:
What happens if another deploy is in progress?

Reply:
Good point. Watch out for hairy fun with possibly non-thread-safe flags :)